### PR TITLE
fix(core): polling to search in cmdk

### PR DESCRIPTION
**Pull request by EYHN**
This is a temporary solution until https://github.com/toeverything/blocksuite/issues/5668 be solved.

### CODE REVIEW

**Comment by EYHN** [comment]
Current dependencies on/for this PR:
* canary
  * **PR #5274** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5274?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5274?utm_source=stack-comment).

File: packages/frontend/core/src/components/pure/cmdk/data.tsx [hunk 1]
```diff
@@ -216,7 +224,25 @@ export const usePageCommands = () => {
   const navigationHelper = useNavigateHelper();
   const t = useAFFiNEI18N();
 
+  const [searchTime, setSearchTime] = useState<number>(0);
+
+  // HACK: blocksuite indexer is async,
+  // so we need to re-search after it has been updated
+  useEffect(() => {
```
Comment (pengx17) [inline packages/frontend/core/src/components/pure/cmdk/data.tsx:231]: I think making the `query` a debounced value could also implicitly solve the issue.
Also we should make query debounced to improve the searching performance.

Comment (EYHN) [reply packages/frontend/core/src/components/pure/cmdk/data.tsx:231]: The indexer may delay by 1 second,
or the content may not have been synced, which may take longer.

debounce cannot solve this, this is a temporary solution until https://github.com/toeverything/blocksuite/issues/5668 be solved.

Comment (pengx17) [reply packages/frontend/core/src/components/pure/cmdk/data.tsx:231]: Oh I just noticed that this function is essentially a setInterval.

**Review by pengx17 — APPROVED** [review]

**Pull request merged**